postfix/init.sls

```diff
@@ -1,5 +1,16 @@
 {% from "postfix/map.jinja" import postfix with context %}
 
+{%- if grains['os_family']=="Suse" %}
+# The existence of this file prevents the system to
+# overwrite files from salt when installing.
+/var/adm/postfix.configured:
+  file.managed:
+    - contents: ''
+    - mode: '0644'
+    - user: 'root'
+    - group: 'root'
```

Comment:
This state needs a `require_in` (for `pkg: postfix`, right?)

Reply:
Exactly, good catch.